packages/web-pkg/src/helpers/share/link.ts

```diff
@@ -80,6 +80,7 @@ export const copyQuicklink = async (args: CopyQuickLink) => {
 
   const linkSharesForResource = await clientService.owncloudSdk.shares.getShares(resource.path, {
     share_types: ShareTypes?.link?.value?.toString(),
+    spaceRef: `${resource.storageId}!${resource.nodeId}`,
```

Comment:
not possible to use `resource.id` or `resource.fileId`?

Reply:
I checked with Jörn, we think that's not possible. But I can check again

Reply:
You are right, works with fileId, this IDs driving me nuts 🙈